bundles/net.tourbook/src/net/tourbook/ui/views/TourBlogView.java

```diff
@@ -613,8 +606,11 @@ private void create_24_Tour(final StringBuilder sb) {
                                  + " title='" + hoverEdit + "'" //                  //$NON-NLS-1$ //$NON-NLS-2$
                                  + ">" //                                           //$NON-NLS-1$
                                  + "</a>") //                                       //$NON-NLS-1$
-                           + "   </div>" + NL) //                                   //$NON-NLS-1$
-                     + ("<span class='blog-title'>" + tourTitle + "</span>" + NL)); //$NON-NLS-1$ //$NON-NLS-2$
+                           + "   </div>" + NL)); //                                   //$NON-NLS-1$
+
+               if (net.tourbook.common.util.StringUtils.hasContent(tourTitle)) {
```

Comment:
Can you use the flag `isTitle` instead of `net.tourbook.common.util.StringUtils.hasContent(tourTitle)` which already checks the content

Reply:
Good idea, fixed